style.css

```diff
@@ -63,76 +67,48 @@ nav{
   padding: 8px 30px;
 }
 
-#headline h2{
-  color: #172B4D;
+#headline h2 {
+  color: #172b4d;
   font-size: 40px;
   padding: 10px 0;
 }
```

Comment:
- [ ] Kindly here use `h1` instead of `h2` because this is the main header, and make it bigger to match the design.
- [ ] Kindly add some padding-top to match the Figma design.

Your design
![image](https://user-images.githubusercontent.com/35535130/158388942-88d00ea1-2322-4a7c-9c5f-b9b416c53ef9.png)

Original design
![image](https://user-images.githubusercontent.com/35535130/158389068-b15bdfab-30d3-4733-a068-17ba8c3c9802.png)